docs/ensnode.io/config/integrations/starlight.ts

```diff
@@ -149,7 +149,9 @@ export function starlight(): AstroIntegration {
       replacesTitle: true,
     },
     social: {
+      twitter: "https://x.com/NamehashLabs",
       github: "https://github.com/namehash/ensnode",
+      telegram: "https://t.me/namehash",
```

Comment:
Let's use official ENSNode tg channel:
```suggestion
      telegram: "https://t.me/ensnode",
```